app/views/layouts/embed_layout.html.haml

```diff
@@ -23,18 +23,18 @@
           .alert.alert-error
             %a.close{"data-dismiss" => "alert"} ×
             %h4
-              Please fix the following 
-              = pluralize(@b_param.bike_errors.count, "error")
+              = t(".please_fix_the_following")
+              = pluralize(@b_param.bike.errors.count, t(".error"))
```

Comment:
Oops, typo. Fixed in f6551a4efe046d4979dd547aeb3953544c6a62e7

Reply:
Damn, missed that. Thanks!